Engine/src/ECS/Systems/RenderSpriteSystem.bf

```diff
@@ -6,8 +6,6 @@ namespace SteelEngine.ECS.Systems
 {
 	public class RenderSpriteSystem : BaseSystem
 	{
-		public this(Application app) : base(app) {}

```

Comment:
I think that is fine! No reason to allow more than one Application I think. As long as we don't think this will end potential support for more than one Window.